resources/assets/lib/chat/chat-api.ts

```diff
@@ -68,6 +68,21 @@ export default class ChatAPI {
     });
   }
 
+  partChannel(channelId: number, userId: number) {
+    return new Promise((resolve, reject) => {
```

Comment:
`$.ajax` has `.then` and `.catch` since 3.0 so the wrapping isn't needed anymore.

Reply:
noted